template/schannel.admx

```diff
@@ -319,7 +325,6 @@
                         </item>
                     </trueList>
                     <falseList defaultKey="SYSTEM\CurrentControlSet\Control\SecurityProviders\SCHANNEL\Protocols\TLS 1.1\Server">
-                        <item valueName=""><value><decimal value="" /></value></item>
```

Comment:
Is this the fix for #10? If so, you could add "Closes #10" to the PR description?

Reply:
Not a confirmed fix, since I haven't encountered the issue myself and so I can't diagnose/troubleshoot. I would say possible fix.

Reply:
Hopefully sheehand can confirm.